src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -69,20 +67,10 @@
 
     private Long maxBillingBytes;
 
-    private boolean useQueryApi;
-
     private final Set<BQStatementRoot> runningStatements = Collections.synchronizedSet(new HashSet<BQStatementRoot>());
 
-    /** Boolean to determine, to use or doesn't use the ANTLR parser */
-    private boolean transformQuery = false;
-
-    /** getter for transformQuery */
-    public boolean getTransformQuery() {
-        return transformQuery;
-    }
-
-    /** Boolean to determine whether or not to use legacy sql (default: true) **/
-    private boolean useLegacySql = true;
+    /** Boolean to determine whether or not to use legacy sql (default: false) **/
+    private boolean useLegacySql = false;
```

Comment:
This is kind of a sneaky change.
What do you think ? Why wouldn't non-legacy be the default....

Reply:
The BQ API defaults to useLegacySql=true, so maybe they were matching that?  

>  useLegacySql  
> boolean
> Specifies whether to use BigQuery's legacy SQL dialect for this query. The default value is true. If set to false, the query will use BigQuery's standard SQL: https://cloud.google.com/bigquery/sql-reference/ When useLegacySql is set to false, the value of flattenResults is ignored; query will be run as if flattenResults is false.

Reply:
I'm all for making standard be the default. Especially if we're in here changing supported option (what might be a good SemVer major version bump). I might argue that this should be a different PR. But I'm not opposed.